cfgov/jinja2/v1/home-page/index_new.html

```diff
@@ -1,16 +1,15 @@
 {% extends "layout-full.html" %}
 
 {% import "organisms/card-group.html" as card_group with context %}
+{% import "organisms/carousel.html" as carousel with context %}
 
 {% block title -%}
     {{ _('Consumer Financial Protection Bureau') }}
 {%- endblock %}
 
 {% block content_main %}
 {% block carousel %}
```

Comment:
Not directly related to this PR, but does having a specific named `block` for this serve any purpose? The info unit and card sections don't have their own named blocks.

Reply:
This might be leftover from previous implementations. We can remove if it's not needed.